modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISortRelevantObjectEntryModelListenerTest.java

```diff
@@ -132,6 +132,9 @@ private void _addAPIApplication(
 		throws Exception {
 
 		String apiSchemaExternalReferenceCode = RandomTestUtil.randomString();
+		String path =
```

Comment:
Inline this variable if it is only used once.

Reply:
SF is blaming for that

Reply:
@4lejandrito  I did it but SF blames that it should be in a variable

Reply:
true.